### PR TITLE
Remove checked in libc

### PR DESCRIPTION
**Pull request by Ivan-Velickovic**
This was a hack done to make it easier to develop but turns out the toolchain we use packages newlibc which can be easily used instead.

### CODE REVIEW

**Comment by Ivan-Velickovic** [comment]
Will hold off on merging until we can make sure it does not affect benchmarks (it is not expected to but just want to check).

**Comment by Ivan-Velickovic** [comment]
Finally got the CI passing, there seems to be some differences with the GCC toolchain versions.

While perhaps not as elegant, the inclusion of `newlibc.c` which contains stubs should always work.

File: util/newlibc.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include <microkit.h>
+
+void _close(void)
+{
+    microkit_dbg_puts("NEWLIB ERROR: calling unimplimented _close()");
```
Comment (wom-bat) [inline util/newlibc.c:5]: Please fix spelling: 'unimplemented'

**Review by wom-bat — CHANGES_REQUESTED** [review]

File: util/newlibc.c (hunk 1 above)
Comment (Ivan-Velickovic) [reply util/newlibc.c]: Whoops, done. Added a comment at the top for the purpose of the file as well.

**Review by wom-bat — APPROVED** [review]
LGTM.  Will affect my PR on Makefile refactoring, but I can fix that on Wednesday.

**Pull request merged**